tools/cloner-check/generated_files/appconfig.txt

```diff
@@ -85,9 +85,8 @@ github.com/fleetdm/fleet/v4/server/fleet/ZendeskIntegration	GroupID	int64
 github.com/fleetdm/fleet/v4/server/fleet/ZendeskIntegration	EnableFailingPolicies	bool
 github.com/fleetdm/fleet/v4/server/fleet/ZendeskIntegration	EnableSoftwareVulnerabilities	bool
 github.com/fleetdm/fleet/v4/server/fleet/Integrations	GoogleCalendar	[]*fleet.GoogleCalendarIntegration
-github.com/fleetdm/fleet/v4/server/fleet/GoogleCalendarIntegration	Email	string
-github.com/fleetdm/fleet/v4/server/fleet/GoogleCalendarIntegration	PrivateKey	string
 github.com/fleetdm/fleet/v4/server/fleet/GoogleCalendarIntegration	Domain	string
+github.com/fleetdm/fleet/v4/server/fleet/GoogleCalendarIntegration	ApiKey	map[string]string
```

Comment:
Do we need to amend AppConfig.Copy method to copy the map now?

Reply:
Yes, done.